server/config.json

```diff
@@ -19,8 +19,8 @@
       "limit": "100kb"
     },
     "cors": false,
-    "errorHandler": {
-      "disableStackTrace": false
+    "strongErrorHandler": {
+      "debug": false
```

Comment:
Ignore my comment above, I thought this was in middleware config. I'll delete both comments.